ui/lib/core/addon/components/search-select.js

```diff
@@ -103,6 +109,9 @@ export default Component.extend({
       }
       return;
     }
+    if (this.idKey !== 'id' && this.models.length === 1) {
```

Comment:
previously I had the following on line 107: 
```
   if (modelType.includes('identity') || modelType.includes('oidc/client')) {
        this.set('shouldRenderName', true);
      }
```
but instead decided to assume that if we're passing in `objectKeys` and we always want to render the name in the dropdown, in addition to whatever other attrs

Reply:
Not sure how the statement `if (this.idKey !== 'id' && this.models.length === 1)` maps to "If we're passing in `objectKeys` always render the name"?

Reply:
This is a good question - I wasn't sure how to handle this. My thought was:

Revert to original search select behavior of just showing the id in the dropdown if multiple models are passed in and we're attempting to use a dynamic `idKey` other than `id` since it's unlikely all the models would have the same attr.